setup.py

```diff
@@ -100,19 +115,29 @@ def compilation_help():
               '-Werror=cast-align -Werror=int-conversion ' + \
               '-Werror=incompatible-pointer-types'
     os.environ['CFLAGS'] = cflags
-    sys.argv.remove ('-pygame-ci')
+
+STRIPPED=False
+
+# STRIPPED builds don't have developer resources like docs or tests
+# TODO: mangle CFLAGS and strip debug symbols
+
+if "PYGAME_ANDROID" in os.environ:
+    # test cases and docs are useless inside an APK
```

Comment:
They're useful to some.
Testing on user devices is much easier with tests included.
People developing with pygame to learn might like the docs included.

Reply:
On desktop, this is an *option*, on android, it's default. We can always add an option to disable it and include tests in the apk. The idea is to skip docs when packaging a binary with PyInstaller, not when installing via pip.